### ISSUE TITLE
OsmAnd places integration

### ISSUE THREAD

**Issue by timechips** [MM2-0/Kvaesitso#927]
I do not have enough skill to write a plugin, so I'm posting the idea here. Would be great since you can download maps.

**Comment by Sir-Photch** [comment]
We already have places search, and it also uses OSM?

**Comment by timechips** [comment]
> We already have places search, and it also uses OSM?

yes... [OsmAnd](https://osmand.net/) in not the same as OSM. You can download maps and a lot more. Wouldn't be the places search faster when using a downloaded map, and It would of course work without internet. Since it is one of the most popular open source maps app, I think a lot of people would enjoy the option.

**Comment by Sir-Photch** [comment]
...but OsmAnd is also OSM based. Offline maps are out of scope for a launcher. You might as well favourite your location that you've looked up via Kvaesitso and then tap on the map to open up a maps app (which could as well be OsmAnd / OrganicMaps)

**Comment by timechips** [comment]
would it not be possible (similar to breezy weather) that the launcher would return the data in the app (search result, downloaded tiles)? So it would not query OSM, but query the already on device maps you have downloaded in the app. With that it could also refer to favorites and saved places in the app. And favoriting again more than 100 saved places as you are suggesting would be a daunting task.

**Comment by Sir-Photch** [comment]
Its up to OsmAbd to support this as well

**Comment by timechips** [comment]
Here is an [API](https://osmand.net/docs/technical/osmand-api-sdk/add_mapview) I found that looks like it does the trick (note that I'm very inexperienced). This AIDL API specifically.
